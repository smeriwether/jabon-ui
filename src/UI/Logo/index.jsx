import React from 'react';

const Logo = ({ fill, height, width }) => {
  return (
    <svg width={ width } height={ height } viewBox="0 0 500 500" version="1.1" xmlns="http://www.w3.org/2000/svg" baseProfile="full">
      <g transform="scale(1,-1) translate(0,-500)">
        <path d="M 1005.000000 504.000000 C 1005.000000 780.142375 781.142375 1004.000000 505.000000 1004.000000 C 228.857625 1004.000000 5.000000 780.142375 5.000000 504.000000 C 5.000000 227.857625 228.857625 4.000000 505.000000 4.000000 C 781.142375 4.000000 1005.000000 227.857625 1005.000000 504.000000 Z" transform="scale(0.500000,0.500000) translate(-5.000000,-4.000000)" fill={ fill } opacity="1.000000"></path>
        <path d="M 4.250475 467.509710 L 40.546772 467.509710 L 40.546772 4.546747 L 4.250475 4.546747 Z" transform="scale(0.500000,0.500000) translate(658.000000,288.000000)" fill="#ffffff" opacity="1.000000"></path>
        <path d="M 4.992674 521.836410 L 41.286439 521.836410 L 41.286439 4.163590 L 4.992674 4.163590 Z" transform="scale(0.500000,0.500000) translate(588.000000,237.000000)" fill="#ffffff" opacity="1.000000"></path>
        <path d="M 397.296296 40.843044 L 397.296296 4.546747 L 4.703704 4.546747 L 4.703704 40.843044 Z" transform="scale(0.500000,0.500000) translate(299.000000,288.000000)" fill="#ffffff" opacity="1.000000"></path>
        <path d="M 266.831021 38.220480 L 266.831021 4.664922 L 4.608788 4.664922 L 4.608788 38.220480 Z" transform="scale(0.500000,0.500000) translate(362.000000,217.000000)" fill="#ffffff" opacity="1.000000"></path>
        <path d="M 4.608788 189.371588 L 40.905085 189.371588 L 40.905085 4.186403 L 4.608788 4.186403 Z" transform="scale(0.500000,0.500000) translate(362.000000,231.000000)" fill="#ffffff" opacity="1.000000"></path>
        <path d="M 4.871110 137.880080 L 41.167407 137.880080 L 41.167407 4.546747 L 4.871110 4.546747 Z" transform="scale(0.500000,0.500000) translate(288.000000,288.000000)" fill="#ffffff" opacity="1.000000"></path>
        <path d="M 103.814816 40.880083 L 103.814816 4.583786 L 4.703704 4.583786 L 4.703704 40.880083 Z" transform="scale(0.500000,0.500000) translate(299.000000,385.000000)" fill="#ffffff" opacity="1.000000"></path>
        <path d="M 103.136193 40.836410 L 103.136193 4.540113 L 4.617673 4.540113 L 4.617673 40.836410 Z" transform="scale(0.500000,0.500000) translate(593.000000,718.000000)" fill="#ffffff" opacity="1.000000"></path>
      </g>
    </svg>
  );
};

export default Logo;
