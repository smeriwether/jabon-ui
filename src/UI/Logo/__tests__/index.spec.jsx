import Logo from '../../Logo';

describe('<Logo />', () => {
  const setup = (overrides) => {
    const props = {
      fill: null,
      height: null,
      width: null,
      ...overrides
    };

    const wrapper = shallow(<Logo { ...props } />);

    return { wrapper };
  };

  it('should match the snapshot', () => {
    const { wrapper } = setup({ fill: '#ffffff', height: 50, width: 50 });
    expect(wrapper).toMatchSnapshot();
  });
});
