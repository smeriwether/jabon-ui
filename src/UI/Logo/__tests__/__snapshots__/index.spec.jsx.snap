// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should match the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Logo
    fill="#ffffff"
    height={50}
    width={50}
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "baseProfile": "full",
      "children": <g
        transform="scale(1,-1) translate(0,-500)"
      >
        <path
          d="M 1005.000000 504.000000 C 1005.000000 780.142375 781.142375 1004.000000 505.000000 1004.000000 C 228.857625 1004.000000 5.000000 780.142375 5.000000 504.000000 C 5.000000 227.857625 228.857625 4.000000 505.000000 4.000000 C 781.142375 4.000000 1005.000000 227.857625 1005.000000 504.000000 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(-5.000000,-4.000000)"
        />
        <path
          d="M 4.250475 467.509710 L 40.546772 467.509710 L 40.546772 4.546747 L 4.250475 4.546747 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(658.000000,288.000000)"
        />
        <path
          d="M 4.992674 521.836410 L 41.286439 521.836410 L 41.286439 4.163590 L 4.992674 4.163590 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(588.000000,237.000000)"
        />
        <path
          d="M 397.296296 40.843044 L 397.296296 4.546747 L 4.703704 4.546747 L 4.703704 40.843044 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(299.000000,288.000000)"
        />
        <path
          d="M 266.831021 38.220480 L 266.831021 4.664922 L 4.608788 4.664922 L 4.608788 38.220480 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(362.000000,217.000000)"
        />
        <path
          d="M 4.608788 189.371588 L 40.905085 189.371588 L 40.905085 4.186403 L 4.608788 4.186403 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(362.000000,231.000000)"
        />
        <path
          d="M 4.871110 137.880080 L 41.167407 137.880080 L 41.167407 4.546747 L 4.871110 4.546747 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(288.000000,288.000000)"
        />
        <path
          d="M 103.814816 40.880083 L 103.814816 4.583786 L 4.703704 4.583786 L 4.703704 40.880083 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(299.000000,385.000000)"
        />
        <path
          d="M 103.136193 40.836410 L 103.136193 4.540113 L 4.617673 4.540113 L 4.617673 40.836410 Z"
          fill="#ffffff"
          opacity="1.000000"
          transform="scale(0.500000,0.500000) translate(593.000000,718.000000)"
        />
      </g>,
      "height": 50,
      "version": "1.1",
      "viewBox": "0 0 500 500",
      "width": 50,
      "xmlns": "http://www.w3.org/2000/svg",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <path
            d="M 1005.000000 504.000000 C 1005.000000 780.142375 781.142375 1004.000000 505.000000 1004.000000 C 228.857625 1004.000000 5.000000 780.142375 5.000000 504.000000 C 5.000000 227.857625 228.857625 4.000000 505.000000 4.000000 C 781.142375 4.000000 1005.000000 227.857625 1005.000000 504.000000 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(-5.000000,-4.000000)"
          />,
          <path
            d="M 4.250475 467.509710 L 40.546772 467.509710 L 40.546772 4.546747 L 4.250475 4.546747 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(658.000000,288.000000)"
          />,
          <path
            d="M 4.992674 521.836410 L 41.286439 521.836410 L 41.286439 4.163590 L 4.992674 4.163590 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(588.000000,237.000000)"
          />,
          <path
            d="M 397.296296 40.843044 L 397.296296 4.546747 L 4.703704 4.546747 L 4.703704 40.843044 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(299.000000,288.000000)"
          />,
          <path
            d="M 266.831021 38.220480 L 266.831021 4.664922 L 4.608788 4.664922 L 4.608788 38.220480 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(362.000000,217.000000)"
          />,
          <path
            d="M 4.608788 189.371588 L 40.905085 189.371588 L 40.905085 4.186403 L 4.608788 4.186403 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(362.000000,231.000000)"
          />,
          <path
            d="M 4.871110 137.880080 L 41.167407 137.880080 L 41.167407 4.546747 L 4.871110 4.546747 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(288.000000,288.000000)"
          />,
          <path
            d="M 103.814816 40.880083 L 103.814816 4.583786 L 4.703704 4.583786 L 4.703704 40.880083 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(299.000000,385.000000)"
          />,
          <path
            d="M 103.136193 40.836410 L 103.136193 4.540113 L 4.617673 4.540113 L 4.617673 40.836410 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(593.000000,718.000000)"
          />,
        ],
        "transform": "scale(1,-1) translate(0,-500)",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 1005.000000 504.000000 C 1005.000000 780.142375 781.142375 1004.000000 505.000000 1004.000000 C 228.857625 1004.000000 5.000000 780.142375 5.000000 504.000000 C 5.000000 227.857625 228.857625 4.000000 505.000000 4.000000 C 781.142375 4.000000 1005.000000 227.857625 1005.000000 504.000000 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(-5.000000,-4.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 4.250475 467.509710 L 40.546772 467.509710 L 40.546772 4.546747 L 4.250475 4.546747 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(658.000000,288.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 4.992674 521.836410 L 41.286439 521.836410 L 41.286439 4.163590 L 4.992674 4.163590 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(588.000000,237.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 397.296296 40.843044 L 397.296296 4.546747 L 4.703704 4.546747 L 4.703704 40.843044 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(299.000000,288.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 266.831021 38.220480 L 266.831021 4.664922 L 4.608788 4.664922 L 4.608788 38.220480 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(362.000000,217.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 4.608788 189.371588 L 40.905085 189.371588 L 40.905085 4.186403 L 4.608788 4.186403 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(362.000000,231.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 4.871110 137.880080 L 41.167407 137.880080 L 41.167407 4.546747 L 4.871110 4.546747 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(288.000000,288.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 103.814816 40.880083 L 103.814816 4.583786 L 4.703704 4.583786 L 4.703704 40.880083 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(299.000000,385.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "d": "M 103.136193 40.836410 L 103.136193 4.540113 L 4.617673 4.540113 L 4.617673 40.836410 Z",
            "fill": "#ffffff",
            "opacity": "1.000000",
            "transform": "scale(0.500000,0.500000) translate(593.000000,718.000000)",
          },
          "ref": null,
          "rendered": null,
          "type": "path",
        },
      ],
      "type": "g",
    },
    "type": "svg",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "baseProfile": "full",
        "children": <g
          transform="scale(1,-1) translate(0,-500)"
        >
          <path
            d="M 1005.000000 504.000000 C 1005.000000 780.142375 781.142375 1004.000000 505.000000 1004.000000 C 228.857625 1004.000000 5.000000 780.142375 5.000000 504.000000 C 5.000000 227.857625 228.857625 4.000000 505.000000 4.000000 C 781.142375 4.000000 1005.000000 227.857625 1005.000000 504.000000 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(-5.000000,-4.000000)"
          />
          <path
            d="M 4.250475 467.509710 L 40.546772 467.509710 L 40.546772 4.546747 L 4.250475 4.546747 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(658.000000,288.000000)"
          />
          <path
            d="M 4.992674 521.836410 L 41.286439 521.836410 L 41.286439 4.163590 L 4.992674 4.163590 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(588.000000,237.000000)"
          />
          <path
            d="M 397.296296 40.843044 L 397.296296 4.546747 L 4.703704 4.546747 L 4.703704 40.843044 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(299.000000,288.000000)"
          />
          <path
            d="M 266.831021 38.220480 L 266.831021 4.664922 L 4.608788 4.664922 L 4.608788 38.220480 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(362.000000,217.000000)"
          />
          <path
            d="M 4.608788 189.371588 L 40.905085 189.371588 L 40.905085 4.186403 L 4.608788 4.186403 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(362.000000,231.000000)"
          />
          <path
            d="M 4.871110 137.880080 L 41.167407 137.880080 L 41.167407 4.546747 L 4.871110 4.546747 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(288.000000,288.000000)"
          />
          <path
            d="M 103.814816 40.880083 L 103.814816 4.583786 L 4.703704 4.583786 L 4.703704 40.880083 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(299.000000,385.000000)"
          />
          <path
            d="M 103.136193 40.836410 L 103.136193 4.540113 L 4.617673 4.540113 L 4.617673 40.836410 Z"
            fill="#ffffff"
            opacity="1.000000"
            transform="scale(0.500000,0.500000) translate(593.000000,718.000000)"
          />
        </g>,
        "height": 50,
        "version": "1.1",
        "viewBox": "0 0 500 500",
        "width": 50,
        "xmlns": "http://www.w3.org/2000/svg",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <path
              d="M 1005.000000 504.000000 C 1005.000000 780.142375 781.142375 1004.000000 505.000000 1004.000000 C 228.857625 1004.000000 5.000000 780.142375 5.000000 504.000000 C 5.000000 227.857625 228.857625 4.000000 505.000000 4.000000 C 781.142375 4.000000 1005.000000 227.857625 1005.000000 504.000000 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(-5.000000,-4.000000)"
            />,
            <path
              d="M 4.250475 467.509710 L 40.546772 467.509710 L 40.546772 4.546747 L 4.250475 4.546747 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(658.000000,288.000000)"
            />,
            <path
              d="M 4.992674 521.836410 L 41.286439 521.836410 L 41.286439 4.163590 L 4.992674 4.163590 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(588.000000,237.000000)"
            />,
            <path
              d="M 397.296296 40.843044 L 397.296296 4.546747 L 4.703704 4.546747 L 4.703704 40.843044 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(299.000000,288.000000)"
            />,
            <path
              d="M 266.831021 38.220480 L 266.831021 4.664922 L 4.608788 4.664922 L 4.608788 38.220480 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(362.000000,217.000000)"
            />,
            <path
              d="M 4.608788 189.371588 L 40.905085 189.371588 L 40.905085 4.186403 L 4.608788 4.186403 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(362.000000,231.000000)"
            />,
            <path
              d="M 4.871110 137.880080 L 41.167407 137.880080 L 41.167407 4.546747 L 4.871110 4.546747 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(288.000000,288.000000)"
            />,
            <path
              d="M 103.814816 40.880083 L 103.814816 4.583786 L 4.703704 4.583786 L 4.703704 40.880083 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(299.000000,385.000000)"
            />,
            <path
              d="M 103.136193 40.836410 L 103.136193 4.540113 L 4.617673 4.540113 L 4.617673 40.836410 Z"
              fill="#ffffff"
              opacity="1.000000"
              transform="scale(0.500000,0.500000) translate(593.000000,718.000000)"
            />,
          ],
          "transform": "scale(1,-1) translate(0,-500)",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 1005.000000 504.000000 C 1005.000000 780.142375 781.142375 1004.000000 505.000000 1004.000000 C 228.857625 1004.000000 5.000000 780.142375 5.000000 504.000000 C 5.000000 227.857625 228.857625 4.000000 505.000000 4.000000 C 781.142375 4.000000 1005.000000 227.857625 1005.000000 504.000000 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(-5.000000,-4.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 4.250475 467.509710 L 40.546772 467.509710 L 40.546772 4.546747 L 4.250475 4.546747 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(658.000000,288.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 4.992674 521.836410 L 41.286439 521.836410 L 41.286439 4.163590 L 4.992674 4.163590 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(588.000000,237.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 397.296296 40.843044 L 397.296296 4.546747 L 4.703704 4.546747 L 4.703704 40.843044 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(299.000000,288.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 266.831021 38.220480 L 266.831021 4.664922 L 4.608788 4.664922 L 4.608788 38.220480 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(362.000000,217.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 4.608788 189.371588 L 40.905085 189.371588 L 40.905085 4.186403 L 4.608788 4.186403 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(362.000000,231.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 4.871110 137.880080 L 41.167407 137.880080 L 41.167407 4.546747 L 4.871110 4.546747 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(288.000000,288.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 103.814816 40.880083 L 103.814816 4.583786 L 4.703704 4.583786 L 4.703704 40.880083 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(299.000000,385.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "host",
            "props": Object {
              "d": "M 103.136193 40.836410 L 103.136193 4.540113 L 4.617673 4.540113 L 4.617673 40.836410 Z",
              "fill": "#ffffff",
              "opacity": "1.000000",
              "transform": "scale(0.500000,0.500000) translate(593.000000,718.000000)",
            },
            "ref": null,
            "rendered": null,
            "type": "path",
          },
        ],
        "type": "g",
      },
      "type": "svg",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
